ui-manchette-with-spacetimechart/src/hooks/useManchetteWithSpaceTimeChart.ts

```diff
@@ -134,32 +136,41 @@ const useManchettesWithSpaceTimeChart = (
   }, []);
 
   const computedScales = useMemo(
-    () => getScales(operationalPointsWithPosition, height, isProportional, yZoom),
+    () => getScales(operationalPointsWithPosition, { height, isProportional, yZoom }),
     [operationalPointsWithPosition, height, isProportional, yZoom]
   );
 
   // Memoize the returned object to avoid unnecessary recalculations
   return useMemo(
     () => ({
       manchetteProps: {
-        operationalPoints: computedOperationnalPointsHeight,
+        operationalPoints: operationalPointsWithHeight,
         zoomYIn,
         zoomYOut,
         toggleMode,
-        handleScroll,
         manchetteRef: manchette,
         yZoom,
         isProportional,
       },
       spaceTimeChartProps: {
         operationalPoints: operationalPointsWithPosition,
         spaceScales: computedScales,
+        timeScale: timeWindow / xZoom,
         timeWindow,
```

Comment:
You don't need to return the timeWindow anymore
```suggestion
```

Reply:
done